ops/terraform/modules/resources/s3_pii/templates/iam-writer-policy.json

```diff
@@ -4,23 +4,43 @@ ${jsonencode(
   "Id": "bfd-${env}-${name}-iam-writer-policy",
   "Statement": [
     {
-      "Sid": "BFDPIIS3ReadAll",
+      "Sid": "BFDPIIS3ListBuckets",
       "Effect": "Allow",
       "Action": [
         "s3:ListAllMyBuckets",
         "s3:HeadBucket"
```

Comment:
I think we can remove `s3:HeadBucket`, since AWS claims not to recognize that one. I assume it was deprecated in favor of `s3:ListBucket`. (See https://docs.aws.amazon.com/cli/latest/reference/s3api/head-bucket.html)

Reply:
Good call. `HeadBucket` is an operation that requires `ListBucket` permissions.